spec/article_json/export/apple_news/exporter_spec.rb

```diff
@@ -0,0 +1,13 @@
+describe ArticleJSON::Export::AppleNews::Exporter do
+  subject(:exporter) { described_class.new(elements).to_json }
+
+  let(:json) { File.read('spec/fixtures/apple_news_doc.heading_and_basic_text_parsed.json') }
```

Comment:
In the long run this will be the entire reference document, so I would follow the naming scheme of the other files
```suggestion
  let(:json) { File.read('spec/fixtures/reference_document_exported.apple_news.json') }
```

Reply:
Amended to:
```
let(:json) { File.read('spec/fixtures/apple_news_doc.heading_and_basic_text_parsed.json') }
```
(i.e. following the naming scheme- but this is not the exported doc)